lab_2_retrieval_w_bm25/main.py

```diff
@@ -33,7 +44,18 @@ def remove_stopwords(tokens: list[str], stopwords: list[str]) -> list[str] | Non
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(tokens, list) or not isinstance(stopwords, list) \
+            or not all(isinstance(token, str) for token in tokens) \
+            or not all(isinstance(token, str) for token in stopwords):
+        return None
+
+    lst_without_stops = []
+
+    for i in tokens:
```

Comment:
try to use list comprehension (url: https://www.w3schools.com/python/python_lists_comprehension.asp)

Reply:
mb i didn't understand how to use it properly